packages/nodebrew.rb

```diff
@@ -35,24 +22,25 @@ def self.install
     FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/node", "#{CREW_DEST_PREFIX}/bin/nodejs"
     FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew", "#{CREW_DEST_HOME}/.nodebrew"
     FileUtils.ln_sf "#{CREW_PREFIX}/share/nodebrew", "#{HOME}/.nodebrew"
+
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/bash.d/"
+    @env = <<~EOF
+      # nodebrew bash completion
+      source #{CREW_PREFIX}/share/nodebrew/completions/bash/nodebrew-completion
+    EOF
+    IO.write("#{CREW_DEST_PREFIX}/etc/bash.d/nodebrew", @env)
+
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
+    @env = <<~EOF
+      # nodebrew configuration
+      export PATH="$PATH:$HOME/.nodebrew/current/bin"
+    EOF
+    IO.write("#{CREW_DEST_PREFIX}/etc/bash.d/nodebrew", @env)
```

Comment:
Does `IO.write` append or overwrite?  If it's overwrite then the last `@env` in wins.

Reply:
87c1a8e

Reply:
I think you have the same problem with this change.

Reply:
Why not just do everything in one step?  It's the same file, right?

Reply:
No, wait, I think you want this?
```
IO.write("#{CREW_DEST_PREFIX}/etc/etc.d/nodebrew", @env)
```

Reply:
Thanks <3